### ISSUE TITLE
windows msys2 pacman package install

### ISSUE THREAD

**Issue by ahaoboy** [quickjs-ng/quickjs#247]
Install quickjs-ng by pacman like mujs.

https://packages.msys2.org/package/mingw-w64-x86_64-mujs?repo=mingw64

```bash
pacman -S mingw-w64-x86_64-mujs

pacman -S mingw-w64-x86_64-quickjs-ng
```

**Comment by saghul** [comment]
Hum this looks empty: https://packages.msys2.org/package/mingw-w64-x86_64-quickjs-ng?repo=mingw64

Did I miss anything?

It would be great to have it in the msys2 repos!

**Comment by ahaoboy** [comment]
> Hum this looks empty:

Yes, currently quickjs-ng does not support pacman or apt. If it could be installed using a package manager, it would make the migration of projects dependent on mujs more convenient. Otherwise, using the source code would introduce too much noise.

**Comment by saghul** [comment]
Oh ok I thought you were announcing support for that, not asking for it :-) 

I can take a look, I've packaged things for msys2 in the past.

**Comment by ahaoboy** [comment]
Thanks, I am attempting to migrate some libraries that use mujs to quickjs-ng, so it would be immensely helpful if apt and pacman are supported.

**Comment by mcarans** [comment]
@saghul I'm considering replacing the Spidermonkey Firefox 4 (yes it's that old) in the game Oolite with QuickJS-ng. Since you've made a GitHub Actions workflow to build on MINGW64, UCRT64 etc., is there any chance you'll make a PKGBUILD (eg. based on [this template](https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-PKGBUILD-templates/PKGBUILD.CMake-git.sh)) and [make an official MSYS2 package](https://www.msys2.org/wiki/Creating-Packages/) to allow simple installation of QuickJS in these environments on MSYS2?
